pom.xml

```diff
@@ -181,6 +181,10 @@
       <groupId>ch.obermuhlner</groupId>
       <artifactId>big-math</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.errorprone</groupId>
+      <artifactId>error_prone_annotations</artifactId>
+    </dependency>
```

Comment:
Some of these annotations (like `@CanIgnoreReturnValue`) are now getting added to the generated Immutables, but I'm not sure why. Seems like Immutables will add them if they're on the classpath, but from what I can tell, they've always been on the classpath via Guava. 🤷